flowr/src/main/java/com/fueled/flowr/Flowr.java

```diff
@@ -21,21 +30,25 @@
 public class Flowr implements FragmentManager.OnBackStackChangedListener,
         View.OnClickListener {
 
+    public final static String DEEP_LINK_URL = "DEEP_LINK_URL";//to be used as Bundle key for deep links.
+    public final static String FLOWR_CONFIG = "com.fueled.flowr.FlowrConfigImpl"; //Generated file that contain the path to the DeepLinkHandler class.
+
     private final static String KEY_REQUEST_BUNDLE = "KEY_REQUEST_BUNDLE";
     private final static String KEY_FRAGMENT_ID = "KEY_FRAGMENT_ID";
     private final static String KEY_REQUEST_CODE = "KEY_REQUEST_CODE";
-
     private final static String TAG = Flowr.class.getSimpleName();
     private final FragmentsResultPublisher resultPublisher;
     private final int mainContainerId;
     private FlowrScreen screen;
     private ToolbarHandler toolbarHandler;
     private DrawerHandler drawerHandler;
     private Fragment currentFragment;
-
+    private Intent deepLinkIntent;
```

Comment:
@julien-fueled I think this should probably go inside `TransactionData` as it's related to transactions.